packages/asc-ui/src/components/Typography/TypographyStyle.ts

```diff
@@ -36,11 +40,11 @@ export default styled.p<Props>`
       font-weight: 700;
       color: ${themeColor('tint', 'level7')};
     `}
-  ${({ color }) =>
-    color &&
-    css`
-      color: ${themeColor(color)};
-    `}
+    ${({ color }) =>
+      color &&
+      css`
+        color: ${color};
```

Comment:
Would this not break the existing functionality? It seems to me that it should still work the same as before, just simply being deprecated.

Reply:
Makes sense. I was doing two things at the same time.. I will keep the original and make it deprecated